yaml/ver10/cloudintegration/yaml/cloudintegration.yaml

```
@@ -0,0 +1,163 @@
+openapi: 3.1.1
+info:
+  title: Cloud Integration Specification
+  contact:
+    name: ONVIF Technical Specification
+    url: https://github.com/onvif/specs
+    email: help@onvif.org
+  license:
+    name: ONVIF Contributor License Agreement
+    url: https://github.com/onvif/specs/blob/development/LICENSE.md
+  version: "25.06"
+
+components:
+  schemas:
+    Base64DERencodedASN1Value:
+      description: This schema is the equivalent of tas:Base64DERencodedASN1Value in the Security Service specification.
+      type: string
+      format: byte
+      examples: [ Base64DERencodedASN1Value ]
+
+    EndpointReferenceType:
+      description: This schema is the equivalent of wsa:EndpointReferenceType in the WA-Addressing 1.0 specification.
+      type: string
+      format: uri
+
+    AuthorizationServerConfiguration:
+      type: object
+      description: This schema is the equivalent of tas:AuthorizationServerConfigurationData
+      properties:
+        ServerURI:
+          description: Authorization server address
+          type: string
+        ClientID:
+          description: Client identifier issued by the authorization server
+          type: string
+        Scope:
+          description: The requested access scope
+          type: string
+        AuthorizationServerTrustAnchor:
```

Since this is a collection of trust anchors, feels like it should be plural:
```suggestion
        AuthorizationServerTrustAnchors:
```